dashboard/usage.md

```diff
@@ -0,0 +1,112 @@
+---
```

Comment:
the file location is incorrect? @crazycs520

Reply:
great, done.